sys/include/net/gnrc/lwmac/lwmac.h

```diff
@@ -292,6 +292,30 @@ extern "C" {
 #endif
 /** @} */
 
+/**
+* @brief Default message queue size to use for the LWMAC thread.
+*
+* The value of this macro should be enough for supporting the manipulation of
+* LWMAC.
+*
+*/
+#ifndef GNRC_LWMAC_IPC_MSG_QUEUE_SIZE
+#define GNRC_LWMAC_IPC_MSG_QUEUE_SIZE        (8U)
```

Comment:
This is not used anywhere, so why introduce it?

Reply:
OK, this is inherited from the old codes, guess we don't need `MSG_QUEUE_SIZE` any more, so remove it.

Reply:
removed.